truffle-config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     // development: {
     //   host: "127.0.0.1",     // Localhost (default: none)
     //   port: 7545,            // Standard Ethereum port (default: none)
-    //   network_id: "*",       // Any network (default: none)
+    //   network_id: "5777",       // Any network (default: none)
```

Comment:
🙋‍♂️ Ask, what is the purpose of revising this line?

Reply:
emmmm~
copy & paste